addons/sys_list/stringtable.xml

```diff
@@ -96,6 +96,16 @@
             <Portuguese>Cor: Alternar Idiomas do Babel</Portuguese>
             <Turkish>Babel Dil Değiştirme Rengi</Turkish>
         </Key>
+        <Key ID="STR_ACRE_sys_list_LanguageHintPersistence_DisplayName">
+            <English>Persistent Babel Language Indicator</English>
+            <German>Babel Sprachanzeige bleibt sichtbar</German>
+            <Italian>Indicatore di lingua persistente</Italian>
+        </Key>
+        <Key ID="STR_ACRE_sys_list_LanguageHintPersistence_Description">
+            <English>When enabled, the Babel Language Indicator in the bottom right will not fade out if a language other than the "main language" (the first assigned spokenLanguage) is being spoken.</English>
```

Comment:
```suggestion
            <English>Prevent the Babel Language Indicator in the bottom right to fade out if a language other than the "main language" (the first assigned spoken language) is selected.</English>
```
Same for German and Italian.

Reply:
`Prevent the Babel Language Indicator in the bottom right to fade out`
Should be
`Prevent the Babel Language Indicator in the bottom right from fading out`